packages/figma-utils/README.md

```diff
@@ -1,6 +1,14 @@
-# @sit-onyx/figma-utils
+<p>
+  <a href="https://gruppe.schwarz">
+  <div align="center">
+    <img src="https://raw.githubusercontent.com/SchwarzIT/onyx/main/.github/schwarz-group.svg" width="400px" />
+    </div>
+  </a>
+</p>
 
-Utility functions and CLI for importing data from the Figma API into CSS/SCSS variables.
+# Onyx Figma utils
```

Comment:
```suggestion
# Onyx Figma Utils
```

I think in headlines it's expected to have all words (except of/in/the...) starting uppercase ^^